src/blockchain/handler/swap.ts

```diff
@@ -76,9 +81,9 @@ export default class SwapHandler {
 
         const emitter = new Emitter();
 
-        for (const network of Object.keys(Contracts)) {
+        for (const network of Object.keys(this.contracts)) {
```

Comment:
use for `(const network in this.contracts)` - should be more optimized

Reply:
Not more optimized, just syntax sugar.